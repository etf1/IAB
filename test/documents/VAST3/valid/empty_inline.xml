<!-- An empty Inline ad -->
<VAST version="3.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="vast.xsd">
    <Ad>
        <InLine>
            <AdSystem>FreeWheel</AdSystem>
            <AdTitle>My empty ad</AdTitle>
            <Impression>http://www.google.fr</Impression>
            <Creatives></Creatives>
        </InLine>
    </Ad>
</VAST>
