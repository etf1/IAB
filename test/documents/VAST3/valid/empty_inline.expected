{
  "version": "3.0",
  "ads": [
    {
      "adType": 1,
      "adTitle": "My empty ad",
      "adSystem": {
        "name": "FreeWheel"
      },
      "impressions": [
        {
          "uri": "http://www.google.fr"
        }
      ],
      "creatives": []
    }
  ]
}
